src/components/subscriptions/not-subscribed/subscribe.vue

```diff
@@ -47,18 +47,35 @@ export default {
         }
     },
     methods: {
-        subscribe(payload){
+        subscribe(payload) {
             this.showForm = false;
             this.showConfirmation = true;
             this.success = payload;
+        },
+        showFormOrGoToPublication() {
```

Comment:
Suggest moving this configuration (i.e. a mapping between book and subscription page link) to a config file (e.g. keys.js).